src/assembly/source/LICENSE

```diff
@@ -313,7 +313,7 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 jQuery v1.12.0 (jquery-1.12.0.min.js) includes sizzle.js (sizzle.min.js)
 which is distributed under the following license (http://sizzlejs.com/):
  
-Copyright jQuery Foundation and other contributors, https://jquery.org/
+Copyright JS Foundation and other contributors, https://js.foundation/
```

Comment:
HI @gweidner. For jquery, I see https://code.jquery.com/jquery-1.12.0.js. Could you point me to a url with the sizzle.js license?